src/routes/give-away/get.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
```

Comment:
Since you're using giveaway as one word everywhere else, please name this folder `giveaway` (i.e. `src/routes/giveaway/get.js`)